pyoozie/transforms.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2017 "Shopify inc." All rights reserved.
+# Use of this source code is governed by a MIT-style license that can be found in the LICENSE file.
+import six
+
+
+MESSAGE_ON_KILL = '${wf:lastErrorNode()} - ${wf:id()}'
+
+
+def serial_layout(env, error_to_kill=MESSAGE_ON_KILL):
```

Comment:
As long as we don't support cycles, we should be able to represent every dependency tree as a sub-optimal chain.